clippy_lints/src/methods/mod.rs

```diff
@@ -32,7 +34,7 @@ use crate::utils::{
     is_copy, is_expn_of, is_type_diagnostic_item, iter_input_pats, last_path_segment, match_def_path, match_qpath,
     match_trait_method, match_type, match_var, method_calls, method_chain_args, paths, remove_blocks, return_ty,
     single_segment_path, snippet, snippet_with_applicability, snippet_with_macro_callsite, span_lint,
-    span_lint_and_help, span_lint_and_sugg, span_lint_and_then, sugg, walk_ptrs_ty_depth, SpanlessEq,
+    span_lint_and_help, span_lint_and_sugg, span_lint_and_then, sugg, walk_ptrs_ty_depth, LimitStack, SpanlessEq,
```

Comment:
```suggestion
    span_lint_and_help, span_lint_and_sugg, span_lint_and_then, sugg,
    walk_ptrs_ty_depth, LimitStack, SpanlessEq,
```
Helps reduce long strings of text in narrow text readers